external/mdal/3rdparty/libplyxx/libplyxx_internal.h

```diff
@@ -115,8 +115,8 @@ namespace libply
     t.quad = 1;
```

Comment:
revert changes to this file and make it as PR in MDAL. probably changes made in QGIS were not ported to MDAL master...